src/apps/Hosting/HostingDialog.js

```diff
@@ -94,8 +94,14 @@ const CreateHostingDialog = React.createClass({
             <Dialog.SidebarSection>
               Hosting allows you to manage, deploy and publish websites using Syncano platform.
             </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Hosting label">
+              Set the name of your hosting. Label field is not unique.
+            </Dialog.SidebarSection>
+            <Dialog.SidebarSection title="Domains">
+              Define the domain names. Each name must be unique within an Instance.
```

Comment:
'Define the domain names' suggest that we have many names for one domain. 
What do you think about just "Define the domains"? not too confusing?